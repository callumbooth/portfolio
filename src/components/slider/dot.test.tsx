import React from "react";
import renderer from "react-test-renderer";
import Dot from "./dot";

it("Dot displays when inactive", () => {
  const component = renderer.create(
    <Dot active={false} count={0} onClick={() => {}} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Dot displays when active", () => {
  const component = renderer.create(
    <Dot active={true} count={0} onClick={() => {}} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
