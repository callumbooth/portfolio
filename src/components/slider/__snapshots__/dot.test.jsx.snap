// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot displays when active 1`] = `
<button
  className="dot active"
  data-target={1}
  id="dot-0"
  onClick={undefined}
/>
`;

exports[`Dot displays when inactive 1`] = `
<button
  className="dot "
  data-target={1}
  id="dot-0"
  onClick={undefined}
/>
`;
